bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -316,13 +321,13 @@ public String getAction() {
                     ComponentUtils.getEncodedPath(resource.getPath() + ".model.json");
             }
         }
-        return getContextPath() + ADOBE_GLOBAL_API_ROOT + FORMS_RUNTIME_API_GLOBAL_ROOT + "/submit/" + getId();
+        return getContextPath() + resourceResolver.map(ADOBE_GLOBAL_API_ROOT + FORMS_RUNTIME_API_GLOBAL_ROOT + "/submit/" + getId());
```

Comment:
this will need to be taken care by any service consuming this

Reply:
I have handled it every where applicable